src/background/services/secrets/SecretsService.ts

```diff
@@ -209,17 +237,14 @@ export class SecretsService {
     throw new Error('Unsupported import type');
   }
 
-  async getActiveAccountSecrets() {
+  async getActiveAccountSecrets(activeAccount: Account) {
```

Comment:
Since we pass the account as a parameter, I think it would make sense to change this method's signature to `getAccountSecrets(account: Account)`. It no longer matters if it is the active account or no :)